do/serverless.go

```diff
@@ -735,6 +774,62 @@ func (s *serverlessService) ListFunctions(pkg string, skip int, limit int) ([]wh
 	return list, err
 }
 
+// DeleteFunction removes a function from the namespace
+func (s *serverlessService) DeleteFunction(name string, deleteTriggers bool) error {
+	err := initWhisk(s)
+	if err != nil {
+		return err
+	}
+
+	if deleteTriggers {
+		ctx := context.TODO()
+		triggers, err := s.ListTriggers(ctx, name)
+
+		if err == nil {
```

Comment:
See previous comment about clarifying the reason for interpreting error as empty list.